react/components/ErrorBoundary.tsx

```diff
@@ -7,32 +7,33 @@ interface Props {
 }
 
 class ErrorBoundary extends React.Component<Props> {
-  state = {
+  private _state = {
```

Comment:
Maybe `public state`? Why did you need to use `private _state`?

Reply:
It was a vscode suggestion, but believe they're the same thing.

Reply:
Will leave with the pattern we already use though.